src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -130,17 +125,13 @@ void EnReeba_Destroy(Actor* thisx, GlobalContext* globalCtx) {
 void func_80AE4F40(EnReeba* this, GlobalContext* globalCtx) {
     f32 frames = SkelAnime_GetFrameCount(&D_060001E4.genericHeader);
     Player* player = PLAYER;
-    s16 absPlayerVel;
 
     SkelAnime_ChangeAnim(&this->skelanime, &D_060001E4, 2.0f, 0.0f, frames, 0, -10.0f);
 
-    absPlayerVel = fabsf(player->linearVelocity);
-    this->unk_278 = (20 - (absPlayerVel * 2));
-
+    this->unk_278 = 20 - (2 * (s16)fabsf(player->linearVelocity));
```

Comment:
not too sure if cast is actually better than the temp i had here, but i dont feel too strongly on it. would do var * 2 instead of the other way around tho

Reply:
back to being a temp